internal/common/mqtt/mqttconnection.go

```diff
@@ -150,3 +150,25 @@ func (client *Client) Publish(message Message, topic string) error {
 	time.Sleep(time.Second)
 	return nil
 }
+
+var messageHandler MQTT.MessageHandler = func(client MQTT.Client, msg MQTT.Message) {
+
+	log.Printf("Topic %s registered.. %s is the payload \n", msg.Topic(), msg.Payload())
```

Comment:
It would be great if we print the log with `Info` level.
```suggestion
	log.Info(logPrefix, "Topic ", msg.Topic(), " registered.. ", string(msg.Payload()), " is the payload")
```

Reply:
Done!

Reply:
Done!

Reply:
Done!!